Cargo.toml

```diff
@@ -280,6 +280,8 @@ chrono = { version = "0.4.38", default-features = false, features = [
 ] }
 clap = { version = "4.5", features = ["wrap_help", "cargo"] }
 clap_complete = "4.4"
+chrono-tz = "0.8.3"
```

Comment:
Is there some special reason for using this version instead of the current `0.10.0`?

Reply:
No reason, this was just how it was in the original pull request, I didn't change the dependencies. I'm sure they can be updated